awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -10,52 +10,57 @@
 
 class ModuleDocFragment(object):
 
-    # Ansible Tower documentation fragment
+    # Automation Platform Controller documentation fragment
     DOCUMENTATION = r'''
 options:
-  tower_host:
+  controller_host:
```

Comment:
I still have hesitations about the use of bare "controller" in variable names, because it's a very overloaded term in the Ansible world. An Ansible controller is a generic concept, and we haven't been vetting this with the wider community, who may balk at the naming. The alternatives which I can see a strong justification for are:

1. `awx_host`
2. `platform_controller_host`

If you want a reason why this is named `tower_host`, I'll tell you it is because these modules were created before AWX existed. That's the only reason why the AWX naming was not used. In the case of the CLI, created after AWX existed, the entrypoint remains `awx` for the downstream, product, version. If you think about this in the Ansible Platform context, this holds up well against env vars like `ANSIBLE_GALAXY_SERVER`.

The option (2) is the same idea but specific enough.

Reply:
I am a huge fan of `awx_host`, it fits our existing infrastructure well and is shorter, which to me is a win.